swan-lake/learn/calling-java-code-from-ballerina.md

```diff
@@ -84,17 +84,18 @@ This section assumes that you have already read [How to Structure Ballerina Code
 
 #### Creating a Ballerina Project
 ```sh
-> ballerina new yaml-project
-Created new Ballerina project at yaml-project
+> ballerina new yaml_project
+Created new Ballerina project at yaml_project
 
 Next:
     Move into the project directory and use `ballerina add <module-name>` to
     add a new Ballerina module.
 ```
 #### Adding a Ballerina Module to Your Project
+Move into the project directory and execute the following command.
 ```sh
 > ballerina add yamlparser
```

Comment:
We dont need to add a module now. `ballerina new` will create a package with the default module. We can continue with that.

Reply:
The example here demonstrates the `output` option of the `bindgen` command. Therefore, I kept this step along with a note that it is optional if the default root module is being used.